### PR TITLE
Rtd

### PR DESCRIPTION
**Pull request by chStaiger**
With that the RTD doc should be complete.

### CODE REVIEW

**Review by qubixes — CHANGES_REQUESTED** [review]
Nice additions! I have mostly typos for you.

File: docs/source/data_transfers.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
+Data Transfers
+==============
+
+Upload
+------
+To upload data from your local file system to iRODS simply use the `upload` function.
+To determine paths, we recommend to use `pathlib.Path` for local paths and `ibridges.IrodsPath` for iRODS paths.
+
+In the example below we transfer a file or a folder to a new collection *new_coll*. The new collection will be created on the fly. Please note, that this is not true for new nested collections.
```
[the remark this answers is not in the document]
Comment (chStaiger) [reply docs/source/data_transfers.rst]: I changed the wording, gave a hint how to circumvent the error and yes, let's put that on our todo list for the >1.0.0 release.

File: docs/source/data_transfers.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    from ibridges import upload
+    from ibridges import IrodsPath
+    from pathlib import Path
+    
+    local_path = Path(/path/to the/data/to/upload)
+    irods_path = IrodsPath(session, '~', 'new_coll')
+    upload(session, local_path, irods_path)
+
+The new collection will be created on the fly. Please note, that this is not true for new nested collections. Please note that the automatic creation of new collections does not work when they are nested. I.e.
```
[the remark this answers is not in the document]
Comment (chStaiger) [reply docs/source/data_transfers.rst:26]: Done!

File: docs/source/metadata.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+.. code-block:: json
+
+    {'name': '<obj_name>',
+    'irods_id': 24490075,
+    'checksum': 'sha2:XGiECYZOtUfP9lnCGyZaBBkBGLaJJw1p6eoc0GxLeKU=',
+    'metadata': [('Key', 'Value', 'Unit'), ('Key', 'Value', 'Unit1')]}
+
+The dictionary contains the name of the data object or collection and its iRODS identifier.
+Metadata of dataobjects also have an entry for the checksum. The checksum is ot calculated, but extracted from the iRODS database.
```
[the remark this answers is not in the document]
Comment (chStaiger) [reply docs/source/metadata.rst]: Yes, I rephrased!

**Pull request merged**